tekton/ci/jobs.yaml

```diff
@@ -139,21 +139,21 @@ spec:
     - /kaniko/executor
     - $(params.EXTRA_ARGS)
     - --dockerfile=$(params.DOCKERFILE)
-    - --context=$(workspaces.source.path)/$(params.CONTEXT)  # The user does not need to care the workspace and the source.
+    - --context=/workspace/source/$(params.CONTEXT)  # The user does not need to care the workspace and the source.
```

Comment:
this should be $(resources.inputs.source.path) instead, and same in the workingDir

Reply:
yes, and there is something else…